test/allowlist_for_publicAPI.json

```diff
@@ -1225,6 +1225,9 @@
   "torch.multiprocessing.spawn": [
     "Optional"
   ],
+  "torch.nested": [
```

Comment:
Why do you need this since you already have a `__all__` there?

Reply:
TestCorrectModuleNames seemed to fail with just the `__all__`  before adding it here. I'll check again to make sure

Reply:
Double checked this, test_public_bindings.py:test_correct_module_names seems to fail without this